monkey/monkey_island/cc/services/authentication_service/configure_flask_security.py

```diff
@@ -16,10 +16,12 @@
 from .user import User
 
 SECRET_FILE_NAME = ".flask_security_configuration.json"
-AUTH_EXPIRATION_TIME = 30 * 60  # 30 minutes authentication token expiration time
+AUTH_EXPIRATION_TIME = 30 * 60  # 30 minutes
+# Refresh token lives for 30 minutes longer than auth token
+REFRESH_TOKEN_EXPIRATION_DELTA = 30 * 60  # 30 minutes
```

Comment:
This is probably way too long. I think 10% longer is plenty.

Reply:
I don't want to make them co-dependent, but I'll put in 3 minutes